### PR TITLE
Fix "msg": "missing required arg: config_manager_text"

### LINKED ISSUE #86: "msg": "missing required arg: config_manager_text"
I'm getting this error when using the config_manager/load
I don't have that issue when using the config_manager/get?

Following the example of the task to load the config from here: https://github.com/ansible-network/cisco_ios/blob/devel/docs/config_manager/load.md

There is no mention of defining "config_manager_text" anywhere in the doc?

```
  - name: merge interface config to the switch
    include_role:
      name: ansible-network.cisco_ios
      tasks_from: config_manager/load
    vars:
      ios_config_file: "{{ config_file_location }}"

```

```
TASK [ansible-network.cisco_ios : initialize function] ***************************************************************************
included: /opt/src/role_interface_config-access/roles/ansible-network.cisco_ios/includes/init.yaml for my-lab-sw2.gw.me.ca

TASK [ansible-network.cisco_ios : set role basic facts] **************************************************************************
ok: [my-lab-sw2.gw.me.ca]

TASK [ansible-network.cisco_ios : display the role version to stdout] ************************************************************
ok: [my-lab-sw2.gw.me.ca] => {
    "msg": "ansible_network.cisco_ios version is v2.6.4"
}

TASK [ansible-network.cisco_ios : validate ansible_network_os == 'ios'] **********************************************************
skipping: [my-lab-sw2.gw.me.ca]

TASK [ansible-network.cisco_ios : validate ansible_connection == 'network_cli'] **************************************************
skipping: [my-lab-sw2.gw.me.ca]

TASK [ansible-network.cisco_ios : Validate we have required installed version of dependent roles] ********************************
ok: [my-lab-sw2.gw.me.ca]

TASK [ansible-network.cisco_ios : Validate we have required installed version of dependent roles from meta] **********************
skipping: [my-lab-sw2.gw.me.ca]

TASK [ansible-network.cisco_ios : validate config_manager_text is defined] *******************************************************
fatal: [my-lab-sw2.gw.me.ca]: FAILED! => {"changed": false, "msg": "missing required arg: config_manager_text"}
```

I looks like it coming in from "ansible-network.cisco_ios/tasks/config_manager/load.yaml"

```
<....>
- name: validate config_manager_text is defined
  fail:
    msg: "missing required arg: config_manager_text"
  when: config_manager_text is undefined
```

Using:
ansible-galaxy install -r requirements.yml
- ansible-network.network-engine (v2.7.2) is already installed, skipping.
- ansible-network.cisco_ios (stable-2.6) is already installed, skipping.

ansible --version
ansible 2.7.2
  config file = /opt/src/role_interface_config-access/ansible.cfg
  configured module search path = [u'/home/xxxxxx/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Sep 12 2018, 05:31:16) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]

### PR DESCRIPTION
**Pull request by NilashishC**
- Fixes #86 
- Aligns docs and arguments to match other providers

### CODE REVIEW

File: docs/config_manager/load.md [hunk 1]
```diff
@@ -34,84 +34,29 @@ in the `cisco_ios` host group.
...
 ```
 - hosts: cisco_ios
 
   roles:
     - name ansible_network.cisco_ios
       function: config_manager/load
-      config_file: files/ios.cfg
-      replace: yes
+      config_manager_text: "{{ lookup('file', 'ios.cfg') }}"
+      config_manager_replace: yes
```
Comment (samccann) [inline docs/config_manager/load.md]: The text on line 37 says to set this to 'True', but this example sets it to 'yes'.  Would be better to either set this to 'true' here, or change the text to say 'yes' so text and example match.

Comment (NilashishC) [reply docs/config_manager/load.md]: Fixed @samccann

**Review by justjais — APPROVED** [review]

**Review by trishnaguha — CHANGES_REQUESTED** [review]
If `config_manager_file` is passed, we would want to support config_manager_file also.  

In load task:
```
- name: load configuration file contents
  set_fact:
    config_manager_text: "{{ lookup('config_template', config_manager_file) | join('\n') }}"
  when: config_manager_file is defined
```

**Review by trishnaguha — APPROVED** [review]

**Comment by NilashishC** [comment]
@samccann fixed the docs issue you pointed to.

**Review by samccann — APPROVED** [review]
docs portion LGTM

**Pull request merged**